sources/extra_files/app/Core/Frameworks/Baikal/Core/AbstractExternalAuth.php

```diff
@@ -0,0 +1,130 @@
+<?php
```

Comment:
All the patch modifications are from an older PR, https://github.com/YunoHost-Apps/baikal_ynh/pull/24, this part can be ignored for this PR.

Reply:
I am not sure I get it so to be sure, shall I delete these files then ?

Reply:
No, it's only that you took master instead of testing to work on this PR. So, there's so some commit from a previous PR.
I was just saying that we can ignore that part for this PR.